index.bs

```diff
@@ -2778,12 +2778,13 @@ like
 
 <pre><code class="lang-javascript">
   class WritableStream {
-    constructor(underlyingSink = {}, { size, highWaterMark = 1 } = {})
+    <a href="#ws-constructor">constructor</a>(<a href="#ws-constructor">underlyingSink</a> = {}, { size, highWaterMark =
+    1 } = {})
```

Comment:
I forgot that line wrapping is not a good idea inside `<pre>` blocks; sorry, we should undo this.